CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+#👉 How to contribute to Giskard?
+
+Everyone is encouraged to contribute, and we appreciate each and every one of them. Helping the community is thus not limited to writing code. The community will greatly benefit from your efforts to provide clarification, assist others, and enhance the documentation. 📗
+
+Additionally, it is helpful if you help us spread the word by mentioning the library in blog articles about the amazing projects it enabled, tweeting about it on occasion, or just staring the repository! ⭐️
+
+If you choose to contribute, please be mindful to respect our [code of conduct](https://github.com/Giskard-AI/giskard/blob/main/CODE_OF_CONDUCT.md).
+
+## The different ways you can contribute to Giskard!
+
+There are 4 ways you can contribute to Giskard:
+* Fixing outstanding issues with the existing code;
+* Implementing new tests;
+* Contributing to the examples or to the documentation;
+* Submitting issues related to bugs or desired new features.
+
+
+### Did you find a bug? 
+
+First, we would really appreciate it if you could **make sure the bug was not
+already reported** (use the search bar on Github under Issues).
+
+If you did not find it, please follow these steps to inform us:
+
+* Include your **OS type and version**, the versions of **Python**, and different Python libraries you used;
+* A short, self-contained, code snippet that allows us to reproduce the bug in less than 30s;
+* Provide the *full* traceback if an exception is raised.
```

Comment:
I think you mean stacktrace. Not sure what traceback means.

Reply:
Both words are synonyms (https://realpython.com/python-traceback/) and are used the same way.
But ok, let's use stack trace